dot/rpc/modules/chain_test.go

```diff
@@ -388,8 +408,7 @@ func loadTestBlocks(t *testing.T, gh common.Hash, bs *state.BlockState, rt runti
 	}
 
 	// Create Block with fake extrinsics
-	blockBody1 := types.Body{0, 1, 2, 3, 4, 5, 6, 7, 8, 9}
-
+	blockBody1 := common.MustHexToBytes("0xad018400d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d0146d0050619728683af4e9659bf202aeb2b8b13b48a875adb663f449f1a71453903546f3252193964185eb91c482cf95caf327db407d57ebda95046b5ef890187001000000108abcd")
```

Comment:
where is this from? looks like it's a real extrinsic?

Reply:
I used the https://github.com/danforbes/polkadot-js-scripts repository that has a script to create a real extrinsic otherwise the runtime returns an error